src/types/webhooks/createWebhook.ts

```diff
@@ -3,4 +3,5 @@ export interface CreateWebhook {
   name: string;
   /** Image for the default webhook avatar */
   avatar?: string | null;
+  reason?: string;
```

Comment:
add a /** */ also what about the function itself?

Reply:
the function should work?
![image](https://user-images.githubusercontent.com/87189679/151144979-d365afb5-2a0f-4182-a8b0-1dbb31f56356.png)